pyproject.toml

```diff
@@ -245,7 +246,10 @@ filterwarnings = [
     "ignore: Trying to infer the `batch_size` from an ambiguous collection", # pytorch_forecasting
     "ignore: Implicitly cleaning up <TemporaryDirectory",
     "ignore: Call to deprecated class DeepARModel.",
-    "ignore: dropout option adds dropout after all but last recurrent layer"
+    "ignore: dropout option adds dropout after all but last recurrent layer",
+    "ignore: Using padding='same' with even kernel lengths and odd dilation may require a zero-padded copy of the input be created",
+    "ignore: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument",
+    "ignore: Initializing zero-element tensors is a no-op"
```

Comment:
Can't we get rid of this warning?

Reply:
It appear in `nn.Linear(output_dims // 2, output_dims // 4)`, when set `output_dims` param < 1
I don't think we can fix it

Reply:
Which file and line is it? I can't find.
How can we set `output_dims` < 1? Why is this possible?

Reply:
https://github.com/etna-team/etna/blob/0f77eb624337a81db4253627aee884871ff5db0b/etna/libs/tstcc/tc.py#L58

Here `input_dims` is `output_dims` in our global model.
Sorry, I mean `output_dims` < 4